packages/core/src/internal/utils/string.ts

```diff
@@ -8,6 +8,12 @@ are a few ways we can handle this.
 2. pull these functions out into their respective sdks
 3. maybe some 3rd party library can help? */
 
-export const decodeFromBase64 = (base64: string) => atob(base64);
+import {Buffer} from 'buffer/index';
 
-export const encodeToBase64 = (str: string) => btoa(str);
+export const decodeFromBase64 = (base64: string) => {
+  return Buffer.from(base64, 'base64').toString('utf8');
+};
+
+export const encodeToBase64 = (str: string) => {
+  return Buffer.from(str, 'utf-8').toString('base64');
+};
```

Comment:
Is the buffer package only needed for browser compatibility?

Reply:
The buffer package allows:
- encoding/decoding to base64 in a browser
- encoding/decoding to base64 with Node version 14 and 15. I have tested 14.15.0 and 15.0.0 specifically since those are the versions the user reported the issue with.

Reply:
We were using `atob` and `btoa` from the built-in Buffer class for Node.js but the user reported that it didn't work with Node v 14 and 15 which @eaddingtonwhite and I were able to reproduce on our laptop.
@cprice404 also tested on arm/x86 EC2 with those Node versions, and that didn't work either.